misk-hibernate-testing/src/main/kotlin/misk/jdbc/TestDatabasePool.kt

```diff
@@ -32,7 +33,7 @@ val SHARED_TEST_DATABASE_POOL = TestDatabasePool(
  *
  * Thread-safe.
  */
-class TestDatabasePool(
+class TestDatabasePool @Inject constructor(
```

Comment:
I don’t think we wanted this created by Guice?

Reply:
Reverted